packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts

```diff
@@ -20,8 +20,9 @@ import type lng from '@lightningjs/core';
 import Base from '../Base';
 import type { Color, StylePartial } from '../../types/lui';
 
-export type ShadowStyle = {
+type ShadowStyle = {
   alpha: number;
+  animation: lng.types.TransitionSettings.Literal;
```

Comment:
is there a different type for animations, or are we using the word animation to mean transition here?

Reply:
The `animation` style prop is actually referring to a transition: `theme.animation.standardEntrance`. 
```js
this.applySmooth(
  this._Shadow,
  { alpha: this.style.alpha, y: this.style.offsetY, x: this.style.offsetX },
  {
    alpha: [this.style.alpha, this.style.animation],
    y: [this.style.offsetY, this.style.animation],
    x: [this.style.offsetX, this.style.animation]
  }
);
```